src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperExecutionContext.cs

```diff
@@ -37,8 +38,7 @@ internal TagHelperExecutionContext(string tagName, bool selfClosing)
         /// Instantiates a new <see cref="TagHelperExecutionContext"/>.
         /// </summary>
         /// <param name="tagName">The HTML tag name in the Razor source.</param>
-        /// <param name="selfClosing">
-        /// <see cref="bool"/> indicating whether or not the tag in the Razor source was self-closing.</param>
+        /// <param name="tagMode">A mode indicating how the tag was written in the Razor source.</param>
```

Comment:
Going to leave `TagHelperOutput` as is since it's not specifically tied to the Razor source's syntax.